cloud/monitoring.rst

```diff
@@ -4,41 +4,42 @@ Monitoring
 Resources Usage
 ===============
 
-Citus Cloud metrics enable you to get information about your cluster's health and performance. The "Metrics" tab of the Cloud Console provides graphs for a number of measurements, all viewable per node.
+Citus Cloud metrics enable you to get information about your cluster's health and performance. The "Metrics" tab of the Cloud Console provides graphs for a number of measurements, all viewable per node. Below are the metrics, broken into categories, with details about the less obvious ones.
 
 Amazon EBS Volume Metrics
 -------------------------
 
-* Read IOPS. The average number of read operations per second.
-   .. image:: ../images/metric-iops-read.png
-* Write IOPS. The average number of write operations per second.
-   .. image:: ../images/metric-iops-write.png
-* Average Queue Length (Count). The number of read and write operation requests waiting to be completed.
-   .. image:: ../images/metric-queue.png
+IOPS are a unit of measure representing input/output operations per second. The operations are measured in KiB, and the underlying drive technology determines the maximum amount of data that a volume type counts as a single I/O.
```

Comment:
I do not understand this sentence - what is this based on?

Reply:
(I closed this PR, opened a duplicate, and replied to your question there)